languages/rust/exercises/concept/README.md

```diff
@@ -0,0 +1,27 @@
+# Rust concept exercises
+
+The concept exercises are based on this [list of concepts][reference].
+
+## Implemented exercises
+
+These are the concept exercises that have currently been implemented, as well as the concepts they teach and their prerequisite concepts:
+
+| exercise                                                            | concepts                                                            | prerequisites                                                       |
+| ------------------------------------------------------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------- |
+| [`numbers`][concept-exercise-numbers]                               | `integers`<br/>`floats`<br/>`conditionals` | -                                                                   |
+| [`enums`][concept-exercise-enums]                                   | `basic-enums`<br/>`conditionals`                            | `strings`                                                     |
+
+It's only important that it's reasonably easy to _find_ the exercise. It's okay if the name isn't perfect. We **will** iterate on this.
+
+## TODO
+
+Thanks for wanting to contribute to the Rust track's concept exercises! Contributions are very welcome!
+
+To contribute, please find and work on one of the [new exercise issues][issues-new-exercise] or [improve exercise issues][issues-improve-exercise].
+
+[reference]: ../../reference/README.md
+[concept-exercises]: ./concept/README.md
+[concept-exercise-enums]: ./enums/.meta/design.md
+[concept-exercise-numbers]: ./numbers/.meta/design.md
+[issues-new-exercise]: https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Atrack%2Frust+label%3Atype%2Fnew-exercise+label%3Astatus%2Fhelp-wanted
```

Comment:
Currently, that is assumed by this process because status/in-progress and status/help-wanted are implicitly mutually exclusive, however I doubt that is actually enforced by GitHub so you do have a point.

Reply:
Ah, good point. Something like #755 would be pertinent.

Reply:
Examining GitHub search syntax yields `-status/in-progress` as a filter to add, and at least a few issues do have both the help-wanted and in-progress labels, so while it's a few more bytes over the wire, it's trivial to add.